WithSecure/tests/test_security_event_connector.py

```diff
@@ -167,7 +167,7 @@ def test_next_batch_is_empty(trigger):
                 "expires_in": 1799,
             },
         )
-        mock_requests.post(
+        mock_requests.get(
```

Comment:
https://connect.withsecure.com/api-reference/elements#post-/security-events/v1/security-events get method is deprecated ?